pkg/reconciler/knativeeventing/knativeeventing.go

```diff
@@ -85,6 +85,10 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, original *v1beta1.Knative
 		return nil
 	}
 
+	if manifest == nil {
+		return nil
```

Comment:
what about logging to indicate there is no manifest?

Reply:
+1